lib/pbench/agent/results.py

```diff
@@ -192,10 +201,11 @@ def copy_result_tb(self):
                 self.logger.info("File uploaded successfully")
             except Exception:
                 self.logger.exception("There was something wrong with your request")
-                sys.exit(1)
+                return 1
         if not response.status_code == 200:
```

Comment:
Again, `response.ok` is probably better; we're likely more concerned here that it worked rather than precisely which success status code we get. (And, arguably, since the `PUT` creates a resource on the server it ought to be returning 201.)

Reply:
Done